code/modules/mob/living/living_defense.dm

```diff
@@ -281,6 +281,12 @@
 
 	return G
 
+/mob/living/proc/surgery_act(mob/living/user, obj/item/I, datum/surgery/S)

```

Comment:
Does this mean we can do surgery on borgs? 😄

Reply:
![image](https://user-images.githubusercontent.com/15887760/75095058-6b7a0880-5591-11ea-8909-7b4e1cec4cd8.png)

All surgery steps expect atleast a carbon now. It's a mess and I'll have to clean that up though